db/tables/operations/drop.py

```diff
@@ -1,5 +1,23 @@
-from db.connection import execute_msar_func_with_engine
+from db.connection import execute_msar_func_with_engine, exec_msar_func
 
 
 def drop_table(name, schema, engine, cascade=False, if_exists=False):
     execute_msar_func_with_engine(engine, 'drop_table', schema, name, cascade, if_exists)
+
+
+def drop_table_from_schema(table_oid, conn, cascade=False):
```

Comment:
Sorry — as I was looking at this code again when implementing schemas.delete, I noticed another nitpick I wanted to mention...

We had `drop_table` and then you added `drop_table_from_schema`. These function names, when taken together are somewhat confusing. I propose renaming them as follows:

- `drop_table` → `drop_table_via_name`
- `drop_table_from_schema` → `drop_table_via_oid`

Reply:
I prefer `drop_table_from_database`. Keep in mind, the `via_name` function will be removed when we get rid of the relevant REST endpoint and model.